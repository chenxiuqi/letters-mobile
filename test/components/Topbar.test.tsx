import React from "react";
import { Topbar } from "@components";
import renderer from "react-test-renderer";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

const setupShallow = () => {
  const wrapper = shallow(<Topbar />);

  return {
    wrapper,
  };
};

describe("Topbar component", () => {
  test("renders", () => {
    const { wrapper } = setupShallow();
    expect(wrapper).toMatchSnapshot();
  });
});
