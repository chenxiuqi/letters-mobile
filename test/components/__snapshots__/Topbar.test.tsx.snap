// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topbar component renders 1`] = `ShallowWrapper {}`;
